### PR TITLE
Conductor metrics collection

### PR DESCRIPTION
**Pull request by marosmars**
conductor now pushes metrics into fluentD
fluentD processes the metrics and exposes them for prometheus
prometheus already collects metrics from fluentD

* add prometheus and grafana compose file for local deployment

Signed-off-by: Maros Marsalek <mmarsalek@frinx.io>

### CODE REVIEW

**Comment by marosmars** [comment]
Needs this to work:
https://github.com/FRINXio/conductor/pull/17

Grafana dashboard prototype:
image

**Review by KenG98 — APPROVED** [review]
LGTM. Waiting on approval from Symphony before merging (since this changed the fluentd configuration).

**Pull request closed without merging**
